components/card/card.css

```diff
@@ -0,0 +1,241 @@
+    padding: 4px;
+    margin: 0px;
+    border-radius: 50%;
+    height: 3%;
+    cursor: pointer;
+    border: 1px solid rgba(121, 121, 121, 0.0);
+
+}
+
+.card-close-btn:hover{
+    border: 1px solid var(--col-primary);
+    font-weight: 800;
+    color: var(--col-primary);
+    background-color: #F5F3FF;
+    transition: background-color 0.4s ease-in-out;
+}
+
+
+/* card default padding*/
+
+.card-py{
+    padding: 0.8rem;
+}
+
+.card-pl{
+    padding-left: 0.5rem !important;
+}
+
+
+.card-mt-1{
+    margin-top: 1rem;
+}
+.card-mb-1{
+    margin-bottom: 1rem;
+}
+.text-center{
+    text-align: center;
+}
+
+.card-rating-padding{
+    padding-top: 1px;
+    padding-left: 7px;
+    color: var(--text-col-white);
+    font-weight: 550;
+    font-size: 0.875rem;
+}
+/*padding*/
+
+
+/* card header text*/
+
+.card-header-text{
+    color: var(--col-primary );
+}
+
+/* card container width*/
+.width-full{
+    width: 100%;
+}
+
+.width-half{
+    width: 50%;
+}
+
+.ui-card-width-300{
+  width: 18.75rem;
+}
+
+.ui-card-width-500{
+    width: 31.25rem;
+}
+
+.ui-card-width-800{
+    width: 50rem;
+}
+
+.ui-card-width-900{
+    width: 56.25rem;
+}
+
+/* cards types */
+.ui-card-default{
+    border: 1px solid rgb(0,0,0,0.1);
+    height:auto;
+    background-color: #fff;
+    border-radius: 0.4rem;
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 0px 0px 1px;
+    line-height: 1.5rem;
+}
+
+.ui-card-basic{
+    border-radius: 0.625rem;
+    background-color: rgb(245, 245, 245);
+    line-height: 1.5rem;
+    
+}
+
+.ui-card-basic-border{
+    border:1px solid rgb(0,0,0,0.1);
+
+}
+.ui-card-basic-border:hover{
+    border: 1px solid rgb(0,0,0,0.2);
+}
+
+
+.ui-card-img{
+    width: 100%;
+    border-radius: 5px;
+   
+}
+
+
+.ui-card-rating{
+    background-color: var(--col-success);
+    
+}
+
+.star-yellow{
+    color: #ffda6b;
+}
+
+.rating-section{
+    background-color: #E03546;
+    border-radius: 3px;
+}
+
+.ui-card-rating-text{
+    color: var(--text-col-white);
+}
+
+/* card list */
+
+.ui-card-list{
+    border:1px solid rgb(0,0,0,0.1);
+    border-radius:6px;
+    display: flex;
+    justify-content: space-between;
+    background-color: #fff;
+}
+.ui-card-list > .ui-card-list-img > img{
+    border-radius: 8px;
+    height: 8.125rem;
+    width: auto;
+}
+
+.ui-card-list > .ui-card-list-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    padding: 0em;
+}
+
+.ui-card-list > .ui-card-list-button{
+    display: flex;
+    align-items: center;
+}
+
+@media screen and (max-width:680px) {
+    .ui-card-list{
+        width: 100%;
+        border:1px solid rgb(0,0,0,0.1);
+        border-radius:6px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+    }
+    .ui-card-list > .ui-card-list-img > img{
+        border-radius: 8px;
+        height: auto;
+        width: 100%;
+    }
+
+    .ui-card-list > .ui-card-list-content{
+        display: flex;
```

Comment:
Everything looks great except some small thing like us of px instead of rem. Cards are promising and looking awesome. Go ahead and merge the branch.